JellyfishClientSdk/project.json

```diff
@@ -52,7 +55,7 @@
   ],
   "manifest_display_name" : "JellyfishClientSdk",
   "name" : "JellyfishClientSdk",
-  "path" : "/Users/angelikaserwa/Projects/ios-client-sdk/JellyfishClientSdk",
+  "path" : "/Users/karolsygiet/sygiet/work/membrane/ios-client-sdk/JellyfishClientSdk",
```

Comment:
this is weird, if there is a local path maybe this shouldn't be checked in the repo or there is something else wrong?

Reply:
huh the webrtc repo does not have this file +  it is also not present in its gitignore idk what to do about that

Reply:
this looks like mockingbird stuff https://mockingbirdswift.com/spm-package-quickstart they don't recommend gitignoring it so let's keep it for now 🤷‍♀️